### ISSUE TITLE
Don't put the jar into your repo

### ISSUE THREAD

**Issue by Schaka** [KevinTyrrell/FishingBot#1]
You should only keep the source in your respository and add the compiled jar to a tagged release here on GitHub.

**Comment by KevinTyrrell** [comment]
I'm so sorry dude, I never got notified of this! I don't understand what you mean though? Why have two repositories?

**Comment by Schaka** [comment]
You don't have a second repository. You have a project (source only) that can be compiled by any user who knows how (there can be a readme).

Then the binary is distributed through tagging your latest master commit and creating a "release" on GitHub, where you can upload the compiled binary.

**Comment by KevinTyrrell** [comment]
Man I've read this so many times and I still can't understand it. Your first two sentences is what I'm already doing, no? I have a runnable jar file for users who want an `exe`-like experience, then the source code provided incase any others want to play around and compile their own version of my work. 

I don't understand the 'tagging' as release stuff. And the compiled binary? What use is a compiled file for? You can't view the source code with it. All you can do is run the program with it if you knew how those files work. Can you explain in simpler terms? Thank you.

**Comment by Schaka** [comment]
Okay, so how this is basically done:
- your repository only contains source code, but not everyone knows how to compile it
- everyone who wants to submit changes or just keep developing in private can fork it/check it out
- for the average user, who just wants to download and use a program, you compile the jar and upload it to your release (on top of your repository, there's a tab to create releases - through git tags)

Take a look here, it should give you an idea:
https://github.com/Schaka/ArenaLive-TBC/releases

**Comment by KevinTyrrell** [comment]
Sorry again for the late response. I totally understand everything you said now.

It's so easy and I had no idea. Thank you for helping me. Are you sure user's are able to understand where to go to find releases? Should you just link the user to the release page or to your main repository?

[GitHub Creating Releases](https://help.github.com/articles/creating-releases/)

**Comment by Schaka** [comment]
You should probably link to the release page. That way your users never get confused by the source. 
In your readme.md, you can also link to the release page, should anyone ever find their way to the main repository.

**Comment by KevinTyrrell** [comment]
Gotcha. Should I add the readme.md to the release or should I rely on the user to come back to the repository if they are unsure of how to use the program?

**Comment by Schaka** [comment]
Readme.md should be added to the repository. For the release, a readme.txt linking the user back to the repository (for either the Wiki part of GitHub or the readme.md formatted), should be fine.

**Comment by KevinTyrrell** [comment]
Ah okay. Thanks man. I really appreciate it.
